README.md

```diff
@@ -28,6 +28,8 @@ __Note:__ Mac OS X, Linux, and Windows are currently the only supported OSes.
       * ```sudo systemctl stop bluetooth``` (once)
       * ```sudo systemctl disable bluetooth``` (persist on reboot)
 
+If you're using BlueZ 5.14 or later, you may find that running noble *and* bleno at the same time stops you from retrieving the list of services over BLE.
+
```

Comment:
Great news, thanks for trying it out!

Yes, I definitely think we should add this info, maybe also [a link back to the noble read me](https://github.com/sandeepmistry/noble#bleno-compatibility). Do we still need to new part about BlueZ or does it just apply to bleno?